src/components/Dashboard/TodoList.vue

```diff
@@ -1385,10 +1386,12 @@ export default class TodoList extends Vue {
         switch (item.correctedFilingType) {
           case FilingTypes.ALTERATION:
           case FilingNames.INCORPORATION_APPLICATION:
+          case FilingNames.CHANGE_OF_COMPANY_INFO:
```

Comment:
I hadn't realized CHANGE_OF_COMPANY_INFO was a type of alteration. Could it be worth changing the name to ALTERATION_OF_COMPANY_INFO to prevent this error in the future?

Reply:
First, I'm not sure what kind of filing this is, nor how we initialize it (ie, not like existing alteration).

Second, that's a question for UI team, since that enum currently maps to "Alteration". I know the business is particular about filing names, but they may be open to change in order to improve the user experience.

Reply:
Third, if Change Of Company Information is a valid filing that we want to correct, then you'll need to update some code in FilingHistoryList.vue.

Note that the BE isn't implemented to support this type of correct at the moment: https://github.com/bcgov/business-schemas/blob/675fe5de5a4d7584a88eb1e8ddfef9889c5da780/src/registry_schemas/schemas/correction.json#L29

I'm thinking that handling this type of filing, and its correction, is future work. Check with Mihai.

Reply:
Note that the BE isn't implemented to support this type of correct at the moment: https://github.com/bcgov/business-schemas/blob/675fe5de5a4d7584a88eb1e8ddfef9889c5da780/src/registry_schemas/schemas/correction.json#L29

I'm thinking that handling this type of filing, and its correction, is future work. Check with Mihai.

Reply:
So the issue I was trying to correct, is that when a correction is saved it is actually saved as a CHANGE_OF_COMAPNY_INFO file type and not as an alteration so when the filing is reloaded from a save it isn't considered a valid filing for a edit ui correction. I drilled down a little more and I think it might actually be an issue with how the filing is saved. I'll look into this more.

Reply:
A correction should be saved as a "correction".

Reply:
The description below should resolve this issue.